ScoutSuite/providers/azure/rules/findings/network-security-groups-rule-inbound-ssh.json

```diff
@@ -0,0 +1,13 @@
+{
+    "dashboard_name": "Network",
+    "description": "Security rule allowing SSH inbound access in security group",
+    "rationale": "You should not permit SSH(port 22) inbound access to a network security group (CIS 6.2).",
+    "path": "network.network_security_groups.id.security_rules.id",
+    "display_path": "network.network_security_groups.id",
+    "conditions": [ "and",
+        ["network.network_security_groups.id.security_rules.id.destination_ports", "containAtLeastOneOf", "22"],
+        ["_INCLUDE_(conditions/exposed-to-the-internet.json)", "", ""],
+        ["_INCLUDE_(conditions/allow-tcp.json)", "", ""]
+    ],
+    "id_suffix": "security_groups_rule_inbound_ssh"
+}
```

Comment:
It seems like those three rules are very similar, and the only difference is the port, the protocol, and the CIS number. I suggest grouping this all in one rule and using arguments (`__ARG_0__`).

Reply:
Not sure if I'm missing something, but I don't see how I can make separate rules using this, it only takes the last defined one